frontend/src/main/pages/Help/HelpCsvPage.jsx

```diff
@@ -152,6 +152,44 @@ export default function HelpCsvPage() {
               </table>
             </Accordion.Body>
           </Accordion.Item>
+          <Accordion.Item eventKey="3">
+            <Accordion.Header>Teams (by Email)</Accordion.Header>
```

Comment:
@jetbrains-junie I wonder if we could put the CSV format for teams into it's own section, with a separate header "Team Information" and a separate accordian?  It shouldn't be grouped with the CSV upload formats.   That's a section for uploads for roster students, which this is NOT.